presto-main-base/src/main/java/com/facebook/presto/sql/planner/plan/IndexJoinNode.java

```diff
@@ -64,7 +66,8 @@ public IndexJoinNode(
                 criteria,
                 filter,
                 probeHashVariable,
-                indexHashVariable);
+                indexHashVariable,
+                lookupVariables);
```

Comment:
What's the lookup variables? Are those used for filter pushdown and non-equal join conditions? Thanks!